.github/ISSUE_TEMPLATE/bug_report_or_issue_report.yml

```diff
@@ -0,0 +1,87 @@
+name: Bug Report or Issue Report
+description: File a bug report or an issue report
+labels: "needs-triage"
```

Comment:
@chipitsine 
Issue form requires at least one default label. I choose "needs-triage" for the default label.
Could you add "needs-triage" label?
https://github.com/SoftEtherVPN/SoftEtherVPN/labels?page=1&sort=name-asc

Reply:
<img width="317" alt="image" src="https://github.com/SoftEtherVPN/SoftEtherVPN/assets/2217296/1b2ee98f-fb57-495f-8f0c-a753af3d9e1b">

Reply:
Thanks!